README.md

```diff
@@ -144,6 +144,27 @@ prpl-server trusts [`X-Forwarded-Proto`](https://developer.mozilla.org/en-US/doc
 
 You should always use `--https-redirect` in production, unless your reverse proxy already performs HTTPS redirection.
 
+## Caching
+
+By default, prpl-server sets the [`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) header to `max-age=60`, except for the entrypoint which gets `max-age=0`. [`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) headers are also sent, so resources that have not changed on the server can be re-validated efficiently.
+
+Choosing `max-age` is a balance. `0` is too low, because HTTP/2 Pushed resources will be considered immediately stale and not actually used, even within the same page load. `3600` is too high, because an update to the application will create an hour-long window where some users see the new version, some the old, and others a mixture of both. See [Caching best practices & max-age gotchas](https://jakearchibald.com/2016/caching-best-practices/) for more background.
```

Comment:
This is very prescriptive language. May want to dial it back a little since there's a lot of fine-grain configurations that different developers will want here.

Reply:
Removed this paragraph and cleaned up some other language.